torchvision/transforms/transforms.py

```diff
@@ -156,12 +156,8 @@ def __init__(self, mean, std, inplace=False):
         self.inplace = inplace
 
     def __call__(self, tensor):
-        """
-        Args:
-            tensor (Tensor): Tensor image of size (C, H, W) to be normalized.
-
-        Returns:
-            Tensor: Normalized Tensor image.
+        """Apply the transform to the given tensor (C x H x W) and return the
```

Comment:
nit: Please make changes that are not related to the particular goal of the PR in a separate PR.

Reply:
Thanks for getting back to me. I'll address these right away.

Reply:
Fixed with 17db875.